zenodo/modules/deposit/views.py

```diff
@@ -259,6 +260,18 @@ def current_datetime():
         'current_time': now.time(),
     }
 
+@blueprint.route(
+    '/record/extractmetadata/<key>/<version_id>',
+    methods=['POST']
+)
+def extractmetadata(key=None, version_id=None):
+    """Extract metadata for a given file
+
+    NOTE:
+    It should probably go to `invenio-files-rest` module or
```

Comment:
I think the endpoint will go to separate module to keep it simple in the invenio-files-rest